native/packages/react-native-bpk-component-card/src/BpkCard.commontests.js

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
In other components, the filename is `componentName-test.common.js`. I think the filename here should change to fit that convention.

Reply:
👍